cytomineprojectmigrator/exporter.py

```diff
@@ -160,7 +162,9 @@ def _download_image(image, path):
             self.save_user(user, "userannotation_creator")
 
         logging.info("4.2/ Export user annotation term creator users")
-        annotation_users = set([annotation.userTerm for annotation in user_annotations if annotation.userTerm])
+        annotation_users = set([
+            annotation.userByTerm[0]['user'].pop() for annotation in user_annotations if annotation.userByTerm
+        ])
```

Comment:
When set, `userByTerm` is a list of dictionaries with the following keys:
* `id`: the identifier of the link between a term and a link
* `term`: the identifier of the term
* `user`: a list of user identifiers that linked that term to the annotation.
An annotation can be associated to multiple terms, and distinct users can associate the same term to the annotation (while this use-case is very rare).

Documentation is missing but behavior can be derived from server implementation (https://github.com/Cytomine-ULiege/Cytomine-core/blob/master/grails-app/services/be/cytomine/ontology/AnnotationListingService.groovy#L188)

Why only checking the first item in the list and the last user ? The goal here is to get all users involved in these annotation/term association things.